### ISSUE TITLE
Fights take too long: damage should be increased

### ISSUE THREAD

**Issue by magnus-ISU** [lxgr-linux/pokete#197]
**Is your feature request related to a problem? Please describe.**
It's very annoying to fight poketes at the moment (considering this is the main system in the game, this is not good!). I applied a personal hotfix by speeding up the game a ton, but that's not a real solution. I think the better alternative would be to make attacks have much higher damage so combats end in fewer exchanges.

**Describe the solution you'd like**
I don't know what the exact formula for damage is, but it should be increased. Right now it feels like tackles do 1-2 damage most of the time. I'd like to see them deal at least 4 to a pokete of equal level or defense or whatever to you. Heavier attacks should be increased accordingly too. This should be both for the player and hostile poketes.

Additionally, it would be cool if poketes in general had higher power levels, especially on their individual nonstandard moves. Right now, except my gobost I use most of my poketes the same way.

**Describe alternatives you've considered**
Reducing health would be worse - another problem is that numbers on attacks are currently too small for them to differentiate themselves from each other.

**Additional context**
It might be worth looking into what other games have done to solve this problem, what they use to vary things up. One core issue with the gameplay is that all fights are 1 on 1, so tactical considerations are hard to make. And items are not reusable (at least so far for me) so I never use them since money is hard to come by.

**Comment by lxgr-linux** [comment]
Yes, buffing some attacks may be a good idea, and 2v2 battles may be possible in the future, due to the restructuring of the fight function.

**Comment by hugojmaia** [comment]
The option to disable battle animations could go a long way towards this.

**Comment by lxgr-linux** [comment]
Yes, incorporating the battle animations into the animations setting can also help here.
But I will look at nerfing/buffing anyways.
